projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -194,6 +195,10 @@ export class CollectionService<S extends EntityState<EntityType, string>, Entity
 
     const storeName = getStoreName(this.store, syncOptions);
 
+    if (syncOptions.reset) {
```

Comment:
Could you add it to syncCollectionGroup also ?
Does it make sense to add it to `syncDoc` ?

Reply:
You're right, I added the reset Option to syncCollectionGroup, syncManyDocs and to syncDocs.